integration/v3_metrics_test.go

```diff
@@ -0,0 +1,142 @@
+// Copyright 2016 The etcd Authors
```

Comment:
can you please move this test to clientv3/integration?

/integration is for server side testing.